docs/outlook/on-new-compose-events-walkthrough.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Automatically set the subject of a new message or appointment
+description: Learn how to implement an event-based add-in that automatically sets the subject of a new message or appointment.
+ms.date: 12/19/2023
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Automatically set the subject of a new message or appointment
+
+Need to insert a required disclaimer into all your messages and appointments? With an event-based add-in, the content is automatically added to new messages or appointments, so that you can focus on composing your message and still remain compliant with your organization's policies.
+
+The following sections walk you through how to develop an add-in that handles the `OnNewMessageCompose` and `OnNewAppointmentOrganizer` events. By the end of this walkthrough, you'll have an add-in that automatically sets the subject of new messages and appointments being created.
```

Comment:
Consider this, in case "walk" gets localized strangely.
```suggestion
The following sections teach you how to develop an add-in that handles the `OnNewMessageCompose` and `OnNewAppointmentOrganizer` events. By the end of this walkthrough, you'll have an add-in that automatically sets the subject of new messages and appointments being created.
```